### ISSUE TITLE
Domain group page: user can create a new domain group

### ISSUE THREAD

**Issue by vickyszuchin** [cisagov/manage.get.gov#2433]
### Issue description

Domain groups are meant to act like a label on a domain. These will be set by admin users to "group" domains together to help with their own internal organization. For instance, some users may group domains that have a particular manager or who expire within the same month. Just like in github where we can filter by a label, the goal is to let users filter by any set of domain groups (labels).

A feature of the domain group page is to allow the user to create a new domain group on the Domain group page.

### Acceptance criteria

- [ ] User can create a create a new domain group 
- [ ] Implementation should conform to fisma design

### Links to other issues

Blocked by:
- #2396

**Comment by katypies** [comment]
Closing this. We don't currently have a sense of whether domain groups will be needed. We can re-open or create a new issue if this comes up in our user interviews or usability testing. I suspect that suborgs may solve many of the intended use-cases here, but it's worth waiting to determine if this is a useful addition in the future.
